switch_model/utilities/__init__.py

```diff
@@ -65,7 +95,7 @@ def create_model(module_list=None, args=sys.argv[1:]):
     create_model(module_list, args=[])
 
     """
-    model = AbstractModel()
+    model = _AbstractModel()
```

Comment:
I am not in favor of using `_` at the beginning of the class. On Python, this means that when we import the module it is going to be a hidden class that we can ignore completely, but I think we should be consistent with the naming of the class

Reply:
Yeah, I could rename it to `CustomAbstractModel`. I used the `_` since it is the convention used elsewhere in switch (e.g. `_ArgumentParser`) however I agree it's not great. Let me know what you prefer!